CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Departments.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+using CheckerUI.Enums;
+using CheckerUI.Helpers.DishesFolder;
+using CheckerUI.Models;
+using CheckerUI.ViewModels;
+
+namespace CheckerUI.Helpers
+{
+    public class Departments : BaseViewModel
+    {
+        private ObservableCollection<DeptModel> m_DeptsList = new ObservableCollection<DeptModel>();
+
+        private List<Dish_item> m_DishItems = new List<Dish_item>();
+
+        public Departments(ObservableCollection<DeptModel> i_list)
+        {
+            m_DeptsList = i_list;
+            GenerateDepts();
+        }
+        private async void GenerateDepts()
+        {
+
```

Comment:
```suggestion
// mock data pulling
```

also, I think this could be outtaken - to be a static method in a utils class dealing with mock data.

Reply:
This is what will be in the end